package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ramda.github.io",
-  "version": "0.0.0",
+  "version": "0.19.1",
```

Comment:
i wonder about this; the version is for the docs site, i don't know if it should track the lib version